### PR TITLE
Sasha/memtier verify only

### PR DESCRIPTION
**Pull request by sashas-lb**

### CODE REVIEW

File: memtier_benchmark.cpp [hunk 1]
```diff
@@ -707,6 +707,9 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
         }
     }
 
+    if (cfg->verify_only && cfg->crc_verify)
+	    cfg->data_verify = 0;
```
Comment (adir-lb) [inline memtier_benchmark.cpp:707]: can you please explain how this code allow verify-only with crc-verify?

File: memtier_benchmark.cpp [hunk 2]
```diff
@@ -710,6 +706,11 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
     if (cfg->verify_only && cfg->crc_verify)
 	    cfg->data_verify = 0;
 
+    if (cfg->crc_verify && !cfg->verify_only && cfg->ratio.is_defined()) {
+	    fprintf(stderr, "error: --ratio and --crc-verify are mutually exclusive.\n");
+            return -1;
```
Comment (adir-lb) [inline memtier_benchmark.cpp]: indentation...

File: memtier_benchmark.cpp (hunk 1 above)
Comment (adir-lb) [reply memtier_benchmark.cpp:707]: btw, what is verify only mode?

Comment (sashas-lb) [reply memtier_benchmark.cpp:707]: verify only mode assumes that previously verify was run .. verify only will read all keys and verify data integrity on those keys

**Pull request merged**